docs/notebooks/channel.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>"Run" → "Call the"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HinodeXRT/xrtpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>